README.md

```diff
@@ -206,6 +215,12 @@ so any changes are thrown away after you exit qemu.  To exit, type
 ### Hacking
 ---
 
+#### Building the cosa container image locally
+---
+
+To completely rebuild the COSA container image locally, use e.g.
+`$ podman build -t localhost/coreos-assembler .`
```

Comment:
maybe mention setting the `$COREOS_ASSEMBLER_CONTAINER` variable if you are running using the bash alias